targets/targets.json

```diff
@@ -8609,7 +8609,7 @@
     "ARM_FM": {
         "inherits": ["Target"],
         "public": false,
-        "extra_labels": ["ARM_FM"]
+        "extra_labels": ["ARM_FM","PSA"]
```

Comment:
Adding the PSA label requires that the target supports KVStore, due to how PSA storage is currently implemented in Mbed OS. Is KVStore supported with fast models?

Reply:
Yes, KVStore is supported by FastModels